migration/data.rst

```diff
@@ -0,0 +1,71 @@
+
+All tables that you wish to migrate must have primary keys. The corresponding destination tables must have primary keys as well, the only difference being that those keys are allowed to be composition to contain the distribution column as well, as described in :ref:`mt_schema_migration`.
+
+Also be sure to distribute tables across the cluster prior to starting a warp so that the data doesn't have to fit on the coordinator node alone.
+
+2. Enable logical replication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some hosted databases such as Amazon RDS require enabling replication by changing a server configuration parameter. On RDS you will need to create a new parameter group, set ``rds.logical_replication = 1`` in it, then change set the parameter group as the active one. Applying the change requires a database server reboot, which can be scheduled for the next maintenance window.
+
+3. Open access for network connection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the Cloud console, identify the hostname (it ends in ``db.citusdata.com``). Dig the hostname to find its IP address:
+
+.. code-block:: bash
+
+  dig +short <hostname> A
+
+On RDS, edit the inbound database security group to add a custom TCP rule:
+
+Protocol
+  TCP
+Port Range
+  5432
+Source
+  <citus ip>/32
+
+This white-lists the IP address of the Citus coordinator node to make an inbound connection. An alternate way to connect the two is to establish peering between their virtual private clouds. We can help set that up if desired.
+
+4. Freeze schema changes
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Put a freeze on the structure of the source database, telling application developers not to modify it. Mismatched table columns/types can confuse the replication process.
+
+5. Run the Warp
+~~~~~~~~~~~~~~~
+
+Contact us and a Cloud engineer will connect to your database with Citus Warp to create a basebackup, open a replication slot, and begin the replication. We can include/exclude your choice of tables in the migration.
+
+During the first stage, creating a basebackup, the Postgres write-ahead log (WAL) may grow substantially if the database is under write load. Make sure you have at least 20% free space on disk during this process. Once the backup is complete and replication begins then the database will be able to shrink the WAL again.
+
+When the replication has caught up with the current state of the source database, the application is free to connect to the new database instead. Any stray updates from the source database will continue to be reflected in the destination.
+
+Finally, when the application is pointing to the new database and no further changes are happening on the source database, contact us again to remove the replication slot. The migration is complete.
```

Comment:
I realize I missed a step in our chat today.

Due the the nature of the replication process, sequence values don't get updated correctly on the destination databases. In order to have the correct sequence value for e.g. an id column, you need to manually adjust the sequence values before turning on writes on the destination database. @saicitus should have a script for that, I believe.